### DISCUSSION TITLE
Adding this repo to Hacktober Fest

### DISCUSSION THREAD

**Discussion by alihassancods** [fahdi/css-visual-learner#75] [Ideas]
Asalam Alykum @fahdi , what do you think about adding the CSS Visual Learner repository to the Hacktoberfest projects list? I believe it's a great project with room for new features, and it would be a fantastic opportunity for Hacktoberfest participants to contribute.

**Comment by fahdi** [comment]
Why not. Do you think it's good enough?

**Reply by alihassancods** [reply]
Yeah OfCourse, its really good. There can be really good addition of features in this open-source repository. I really do remember the time when in December 2023 I contributed to this, and that contribution taught me a lot.

**Reply by fahdi** [reply]
How do we do that?
